content/en/_index.md

```diff
@@ -48,48 +48,44 @@ cascade:
        <div class="feature-icon d-inline-flex align-items-left justify-content-left fs-2 mb-3">
          <a href="/platform-deep-dive/assets/" tabindex="-1" aria-hidden="true"><img src="/homepage/asset-types.svg" alt="Asset Types" title="Asset Types"></a>
        </div>
-       <h4><a href="/platform-deep-dive/assets/asset-types/" class="custom-link-color">Asset Types</a></h4>
+       <h4><a href="/platform-deep-dive/assets/asset-types/" class="custom-link-color"><b>Asset Types</b></a></h4>
        <p>An asset is a software component of value to be tested, such as a web application or API. Explore the asset types we support.</p>
      </div>
      <div class="feature col">
        <div class="feature-icon d-inline-flex align-items-left justify-content-left bg-gradient fs-2 mb-3">
          <a href="/integrations/" tabindex="-1" aria-hidden="true"><img src="/homepage/integrations.svg" alt="Integrations" title="Integrations"></a>
        </div>
-       <h4><a href="/integrations/" class="custom-link-color">Integrations</a></h4>
+       <h4><a href="/integrations/" class="custom-link-color"><b>Integrations</b></a></h4>
        <p>Integrate third-party apps to streamline your workflows, and configure webhooks to get real-time pentest updates.</p>
      </div>
      <div class="feature col">
        <div class="feature-icon d-inline-flex align-items-left justify-content-left bg-gradient fs-2 mb-3">
          <a href="/product-updates/" tabindex="-1" aria-hidden="true"><img src="/homepage/whats-new.svg" alt="What's New" title="What's New"></a>
        </div>
-       <h4><a href="/product-updates/" class="custom-link-color">What's New</a></h4>
+       <h4><a href="/product-updates/" class="custom-link-color"><b>What's New</b></a></h4>
        <p>We regularly ship updates to the Cobalt platform. Read our monthly release blog to stay on top of the latest news.</p>
      </div>
    </div>
 
-
 <div class="row align-items-md-stretch my-5">
-      <div class="col-md-6">
-        <div class="h-100 p-5 border rounded-lg" style="border-color: #D6E3FD">
-          <h3>Manage Your In-House Pentests</h3>
-          <p class="my-2">{{% beta-label %}}</p>
-          <p>Increase the efficiency and quality of your in-house pentests with the Cobalt Pentest Management Platform. Manage all your pentests in one place to streamline your end-to-end processes and standardize reporting.</p>
-          <a href="/pmp/">Learn more »</a>
-        </div>
+  <div class="col-md-6">
+    <div class="card">
+      <div class="card-content">
+  <h3><b class="title-color">Try our DAST Scanner</b></h3>        
```

Comment:
This is nitpicking - if we're using sentence case here, we should apply that to 'Try engagements' or vice versa with title case

Reply:
I am using title case here though, not sentence case, same as the section blocks above.

Reply:
Oh, i was thinking 'our' needs to be capitalized but that doesn't seem to be the case

Reply:
I just used a converter, you are right - changing real quick.